### PR TITLE
adding pull_request_template

### PR DESCRIPTION
**Pull request by itsyitsy**
Keen for what you think about a pull request template like this?

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Documentation Update (if none of the other choices apply)
+
+## Paste Link to the issue
+
+## Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
```
Comment (PradeepKiruvale) [inline .github/pull_request_template.md:18]: unsure->not sure

don't hesitate to contact us, we are here to help!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
+- [ ] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
+- [ ] I ran cargo fmt as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
```
Comment (makr11st) [inline .github/pull_request_template.md:22]:
```suggestion
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] I used cargo clippy as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
```
Comment (makr11st) [inline .github/pull_request_template.md:23]:
```suggestion
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce to Appium?
```
[the remark this answers is not in the document]
Comment (itsyitsy) [reply .github/pull_request_template.md:7]: Oops

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce to `thin-edge.io`?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```
Comment (didier-wenzek) [inline .github/pull_request_template.md:11]: I would add a 4th entry for breaking changes.

**Comment by mneumann** [comment]
When someone submits a PR (other than a trivial change), it should have a `.github/{PR-number}.md` file based on the `.github/pull_request_template.md`.

File: .github/pull_request_template.md (hunk 5 above)
Comment (itsyitsy) [reply .github/pull_request_template.md:11]: Yep good, i added this point

**Pull request merged**